### PR TITLE
White balancing

### PR DESCRIPTION
**Pull request by Mati-AAU**

### CODE REVIEW

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullPipeline.kt
Comment (Casper-NS) [inline app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullPipeline.kt:77]: Figure out how it binarization plays together with whitebalancing before merge

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/opengl_process_stages/WhitebalanceStage.kt
Comment (Casper-NS) [inline app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/opengl_process_stages/WhitebalanceStage.kt:16]: Potentential for better naming of framebuffers

File: app/src/main/res/raw/saturate_shader.glsl
Comment (Casper-NS) [inline app/src/main/res/raw/saturate_shader.glsl:46]: We dont like commented code, either add or remove it

File: app/src/main/res/raw/whitebalance_shader.glsl
Comment (Casper-NS) [inline app/src/main/res/raw/whitebalance_shader.glsl:5]: Rename to better explain what is what in terms of actual picture and whitebalancing

**Review by Casper-NS — CHANGES_REQUESTED** [review]
Minor changes needed but otherwise lgtm.

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullPipeline.kt
Comment (Casper-NS) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullPipeline.kt:77]: works

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/opengl_process_stages/WhitebalanceStage.kt
Comment (Casper-NS) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/opengl_process_stages/WhitebalanceStage.kt:16]: updated

File: app/src/main/res/raw/saturate_shader.glsl
Comment (Casper-NS) [reply app/src/main/res/raw/saturate_shader.glsl:46]: removed

**Review by Casper-NS — APPROVED** [review]

**Review by S3JER — APPROVED** [review]

**Pull request merged**
